__tests__/pet.test.js

```diff
@@ -0,0 +1,246 @@
+const Pet = require('../src/pet');
+
+describe('constructor', () => {
+  it('returns an object', () => {
+    expect(new Pet('Fido')).toBeInstanceOf(Object);
+  });
+
+  it('sets the name property', () => {
+    const pet = new Pet('Fido');
+    expect(pet.name).toEqual('Fido');
+  });
+
+  it('has an initial age of 0', () => {
+    const pet = new Pet('Fido');
+    expect(pet.age).toEqual(0);
+  });
+
+  it('has an initial hunger of 0', () => {
+    const pet = new Pet('Fido');
+    expect(pet.hunger).toEqual(0);
+  });
+
+  it('has an intial fitness of 10', () => {
+    const pet = new Pet('Fido');
+    expect(pet.fitness).toEqual(10);
+  })
+}); 
+
+describe('growUp', () => {
+  it('increments the age by 1', () => {
```

Comment:
You could have the first 3 it block combined into one, just to save some space, in the description you can add all three conditions, as an explanation what is going to happen.
`````
  it("increments the age by 1, increases hunger by 5 and decreases fitness by 3", () => {
    const pet = new Pet("Fido");

    pet.growUp();

    expect(pet.age).toEqual(1);
    expect(pet.hunger).toEqual(5);
    expect(pet.fitness).toEqual(7);
  });
`````